docs/list.json

```diff
@@ -1,4 +1,4 @@
-const list = {
+{
```

Comment:
Please undo these changes to `list.json`. It seems not right what you have done in this file.